src/Game.java

```diff
@@ -0,0 +1,32 @@
+public class Game {
+
+    Players playerOne;
+    Players playerTwo;
+
+    public Game(Players firstPlayerName, Players secondPlayerName) {
+        this.playerOne = firstPlayerName;
+        this.playerTwo = secondPlayerName;
+        
+    }
+
+    public void buttonWasPressedWithValue(Message mail) {
```

Comment:
Nitpick: we should probably rename this method. Also why is the variable called `mail`? 😄

Reply:
You're right, the name is too long. I don't know why but the name before was more ___stupid___ :)